iFixit/toolbox
# Add box-shadow transition

This adds a transition to the `<SearchInput>` box-shadow.

cr_req 1

## QA 
- Go to https://ifixit.design#searchinput and focus/unfocus the SearchInput. Notice that the box shadow has no transition (the change occurs immediately)
- Assert that when focusing/un-focusing the SearchInput on https://deploy-preview-31--toolbox.netlify.com/#searchinput the box shadow eases in and out. Note that the change here is pretty subtle, but you should notice that the change in box shadow is slightly less jarring.
